### PR TITLE
New WebView2 versioning doc

### PR DESCRIPTION
**Pull request by JoshuaWeber**
New WebView2 concept doc, Versioning.

### CODE REVIEW

**Comment by JoshuaWeber** [comment]
@signoff

File: microsoft-edge/webview2/concepts/versioning.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 05/19/2020
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+keywords: IWebView2, IWebView2WebView, webview2, webview, wpf apps, wpf, edge, ICoreWebView2, ICoreWebView2Host, browser control, edge html
+---
+
+# Browser version needed for WebView2
```
Comment (Reezaali) [inline microsoft-edge/webview2/concepts/versioning.md:13]:
```suggestion
# Understanding browser versions and WebView2  
```

File: microsoft-edge/webview2/concepts/versioning.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+ms.prod: microsoft-edge
+ms.technology: webview
+keywords: IWebView2, IWebView2WebView, webview2, webview, wpf apps, wpf, edge, ICoreWebView2, ICoreWebView2Host, browser control, edge html
+---
+
+# Browser version needed for WebView2  
+
+WebView2 is dependent upon Microsoft Edge to function.  Each WebView2 SDK has a minimum browser version required to be installed.  This browser version is specified in our [Release Notes](../releasenotes.md).  You may see the minimum version reflected in the patch version of the SDK packaged version string \(for example, `SDK package version 0.9.488` indicates a minimum browser build number of 488\).  
+
+To utilize WebView2 in your app, the minimum version of [Microsoft Edge (Chromium)](/microsoft-edge/index) supported by the WebView2 SDK must be installed on your machine.  To access the latest releases of the browser please see [Browser Channels](deployedge/microsoft-edge-channels).
```
Comment (Reezaali) [inline microsoft-edge/webview2/concepts/versioning.md:17]: The browser channels link is broken. Can you review please?

Comment (Reezaali) [inline microsoft-edge/webview2/concepts/versioning.md:17]:
```suggestion
WebView2 depends on Microsoft Edge to function. Each WebView2 SDK requires that a minimum browser version is installed. This browser version is specified in our [Release Notes](../releasenotes.md). You may see the minimum version reflected in the package version of the SDK. For example, if you use the `SDK package version 0.9.488`, then you need to install Microsoft Edge with a build number of 488 or later. For more information on the latest releases of the browser, see [Browser Channels](deployedge/microsoft-edge-channels).
```

File: microsoft-edge/webview2/concepts/versioning.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+---
+
+# Browser version needed for WebView2  
+
+WebView2 is dependent upon Microsoft Edge to function.  Each WebView2 SDK has a minimum browser version required to be installed.  This browser version is specified in our [Release Notes](../releasenotes.md).  You may see the minimum version reflected in the patch version of the SDK packaged version string \(for example, `SDK package version 0.9.488` indicates a minimum browser build number of 488\).  
+
+To utilize WebView2 in your app, the minimum version of [Microsoft Edge (Chromium)](/microsoft-edge/index) supported by the WebView2 SDK must be installed on your machine.  To access the latest releases of the browser please see [Browser Channels](deployedge/microsoft-edge-channels).  
+
+> [!NOTE]
+> WebView2 is currently in Preview.  The Microsoft Edge WebViews team strives to ensure backward compatibility between browser versions and SDKs, it is not guaranteed, however some newer versions of the browser may not support older SDK versions.  If there are breaking changes between browser versions and SDK, the Microsoft Edge WebViews team indicates the changes in the [release notes](../releasenotes.md)
```
Comment (Reezaali) [inline microsoft-edge/webview2/concepts/versioning.md:20]:
```suggestion
> WebView2 is currently in Preview.  While, the Microsoft Edge WebViews team strives to ensure backward compatibility between browser versions and SDKs, it is not guaranteed as some newer versions of the browser may not support older SDK versions.  If there are breaking changes between browser versions and SDKs, the Microsoft Edge WebViews team indicates the changes in the [release notes](../releasenotes.md)  
```

File: microsoft-edge/webview2/concepts/versioning.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+To utilize WebView2 in your app, the minimum version of [Microsoft Edge (Chromium)](/microsoft-edge/index) supported by the WebView2 SDK must be installed on your machine.  To access the latest releases of the browser please see [Browser Channels](deployedge/microsoft-edge-channels).  
+
+> [!NOTE]
+> WebView2 is currently in Preview.  The Microsoft Edge WebViews team strives to ensure backward compatibility between browser versions and SDKs, it is not guaranteed, however some newer versions of the browser may not support older SDK versions.  If there are breaking changes between browser versions and SDK, the Microsoft Edge WebViews team indicates the changes in the [release notes](../releasenotes.md)  
+
+In the future, the Microsoft Edge WebViews plans to change the distribution model.  The Microsoft Edge WebViews plans to removed the direct dependency on Microsoft Edge browser from WebView2 in a future release.  To learn more, see [WebView2 Runtime]() in the [Distribution]() section.
```
Comment (Reezaali) [inline microsoft-edge/webview2/concepts/versioning.md:22]:
```suggestion
In the future, the Microsoft Edge WebViews team plans to change the distribution model.  The Microsoft Edge WebViews team plans to remove the direct dependency on the Microsoft Edge browser from WebView2.  To learn more, see [WebView2 Runtime]() in the [Distribution]() section.  
```

Comment (Reezaali) [reply microsoft-edge/webview2/concepts/versioning.md:22]: Can you review the Distribution link - it's missing?

File: microsoft-edge/webview2/concepts/versioning.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+In the future, the Microsoft Edge WebViews plans to change the distribution model.  The Microsoft Edge WebViews plans to removed the direct dependency on Microsoft Edge browser from WebView2 in a future release.  To learn more, see [WebView2 Runtime]() in the [Distribution]() section.  
+
+## Experimental APIs  
+
+While WebViews2 is a preview, the APIs in the SDK represent the expected APIs at GA.  The Microsoft Edge WebViews team included [experimental APIs](../reference/win32/0-9-488-reference-webview2.md#experimental) in the SDK.  The Microsoft Edge WebViews team is evaluating the functionality of the experimental APIs and requests your feedback on each API before including it as a long term feature.  To send feedback, please use the [WebView feedback repo](https://aka.ms/webviewfeedback).
```
Comment (Reezaali) [inline microsoft-edge/webview2/concepts/versioning.md:26]:
```suggestion
While WebView2 is a preview, the APIs in the SDK are expected to remain the same at GA.  There are [experimental APIs](../reference/win32/0-9-488-reference-webview2.md#experimental) included in the SDK.  Please evaluate the experimental APIs and send your feedback on them using the [WebView feedback repo](https://aka.ms/webviewfeedback).  
```

File: microsoft-edge/webview2/concepts/versioning.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+In the future, the Microsoft Edge WebViews plans to change the distribution model.  The Microsoft Edge WebViews plans to removed the direct dependency on Microsoft Edge browser from WebView2 in a future release.  To learn more, see [WebView2 Runtime]() in the [Distribution]() section.  
+
+## Experimental APIs  
+
+While WebViews2 is a preview, the APIs in the SDK represent the expected APIs at GA.  The Microsoft Edge WebViews team included [experimental APIs](../reference/win32/0-9-488-reference-webview2.md#experimental) in the SDK.  The Microsoft Edge WebViews team is evaluating the functionality of the experimental APIs and requests your feedback on each API before including it as a long term feature.  To send feedback, please use the [WebView feedback repo](https://aka.ms/webviewfeedback).  
+
+### Post GA plans  
+
+After WebViews2 reaches a stable general available state and release the 1.0.0 SDK, The Microsoft Edge WebViews team moves all experimental APIs to a pre-release package.  The pre-release package continues to allow for feedback and insight into the latest features, while the stable release version maintains backward compatibility.
```
Comment (Reezaali) [inline microsoft-edge/webview2/concepts/versioning.md:30]:
```suggestion
After WebView2 reaches a stable general available state and we release the 1.0.0 SDK, The Microsoft Edge WebViews team will move all experimental APIs to a pre-release package. The pre-release package continues to allow for feedback and insight into the latest features, while the stable release version maintains backward compatibility.
```

File: microsoft-edge/webview2/concepts/versioning.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+
+In the future, the Microsoft Edge WebViews plans to change the distribution model.  The Microsoft Edge WebViews plans to removed the direct dependency on Microsoft Edge browser from WebView2 in a future release.  To learn more, see [WebView2 Runtime]() in the [Distribution]() section.  
+
+## Experimental APIs  
+
+While WebViews2 is a preview, the APIs in the SDK represent the expected APIs at GA.  The Microsoft Edge WebViews team included [experimental APIs](../reference/win32/0-9-488-reference-webview2.md#experimental) in the SDK.  The Microsoft Edge WebViews team is evaluating the functionality of the experimental APIs and requests your feedback on each API before including it as a long term feature.  To send feedback, please use the [WebView feedback repo](https://aka.ms/webviewfeedback).  
+
+### Post GA plans
```
Comment (Reezaali) [inline microsoft-edge/webview2/concepts/versioning.md:28]:
```suggestion
### Roadmap
```

File: microsoft-edge/webview2/concepts/versioning.md (hunk 2 above)
Comment (JoshuaWeber) [reply microsoft-edge/webview2/concepts/versioning.md]: I'm not sure the correct way to do this. I swapped back to the full link. @jm-trd-ms Had edited it to be a relative link, but maybe that isn't working. I'm not sure the full details. The full link should work.

**Comment by JoshuaWeber** [comment]
I think that resolves all comments and has sign off from everyone.

File: microsoft-edge/webview2/concepts/versioning.md [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+
+> [!NOTE]
+> WebView2 is currently in Preview.  While, the Microsoft Edge WebViews team strives to ensure backward compatibility between browser versions and SDKs, it is not guaranteed as some newer versions of the browser may not support older SDK versions.  If there are breaking changes between browser versions and SDKs, the Microsoft Edge WebViews team indicates the changes in the [release notes](../releasenotes.md)  
+
+In the future, the Microsoft Edge WebViews team plans to change the distribution model.  The Microsoft Edge WebViews team plans to remove the direct dependency on the Microsoft Edge browser from WebView2.  To learn more, see [WebView2 Runtime]() in the [Distribution](./distribution.md) section.
```
Comment (jm-trd-ms) [inline microsoft-edge/webview2/concepts/versioning.md:20]: Please add the missing links and missing files.

Comment (jm-trd-ms) [reply microsoft-edge/webview2/concepts/versioning.md]: Added inline.

**Comment by jm-trd-ms** [comment]
#sign-off and merged.

**Pull request merged**